airflow/www_rbac/utils.py

```diff
@@ -490,3 +492,30 @@ def is_utcdatetime(self, col_name):
         return False
 
     filter_converter_class = UtcAwareFilterConverter
+
+
+def log_request(f_name, *args, **kwargs):
+    """
+    helper function to actually create the log object and commit it
+    """
+    with create_session() as session:
```

Comment:
Will this commit on exit of session?

Reply:
yep 

https://github.com/Affirm/airflow/blob/1.10.9-affirm/airflow/utils/db.py#L50